### PR TITLE
chore: upgrade to 1.11.0

### PR DESCRIPTION
**Pull request by chungquantin**
Status: ✅ Tested on my local `devnet`, `testnet` => Block production / finalization + submit extrinsic works well. 

- ISSUE: https://github.com/r0gue-io/pop-node/issues/141
- Release: https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-v1.11.0
- Prerequisite: https://github.com/r0gue-io/pop-node/pull/182

### References
- https://github.com/frequency-chain/frequency/pull/2117
- https://github.com/r0gue-io/base-parachain/pull/23
- Compare `pallet-contracts`: [`v1.10.0`](https://github.com/paritytech/polkadot-sdk/commits/polkadot-v1.10.0/substrate/frame/contracts) and [`v1.11.0`](https://github.com/paritytech/polkadot-sdk/commits/polkadot-v1.11.0/substrate/frame/contracts)
- Compare `pallet-xcm`: [`v1.10.0`](https://github.com/paritytech/polkadot-sdk/commits/polkadot-v1.10.0/substrate/frame/xcm) and [`v1.11.0`](https://github.com/paritytech/polkadot-sdk/commits/polkadot-v1.11.0/substrate/frame/xcm)
## Direct changes
- [[#3512](https://github.com/paritytech/polkadot-sdk/pull/3512)]: [FRAME] Introduce Runtime Omni Bencher
- [[#2714](https://github.com/paritytech/polkadot-sdk/pull/2714)]: GenesisConfig presets for runtime
- [[#3801](https://github.com/paritytech/polkadot-sdk/pull/3801)]: Remove `slot_duration` from `aura::collators::basic::Params`
- [[#2292](https://github.com/paritytech/polkadot-sdk/pull/2292)]: Migrate Fee Payment from Currency to fungible traits
- [[#4017](https://github.com/paritytech/polkadot-sdk/pull/4017)]: Complete removal of `TryRuntime` subcommand

### CODE REVIEW

File: runtime/devnet/src/lib.rs [hunk 1]
```diff
@@ -955,12 +942,16 @@ impl_runtime_apis! {
...
+			build_state::<RuntimeGenesisConfig>(config)
+		}
+
+		fn get_preset(id: &Option<sp_genesis_builder::PresetId>) -> Option<Vec<u8>> {
+			get_preset::<RuntimeGenesisConfig>(id, |_| None)
 		}
 
-		fn build_config(config: Vec<u8>) -> sp_genesis_builder::Result {
-			build_config::<RuntimeGenesisConfig>(config)
+		fn preset_names() -> Vec<sp_genesis_builder::PresetId> {
```
Comment (al3mart) [inline runtime/devnet/src/lib.rs:953]: We might be missing the chance to move genesis config from the node to the runtime. For instance as in [here](https://github.com/polkadot-fellows/runtimes/blob/ca6992420f08731b2a683bfa893354b72a15848f/relay/polkadot/src/genesis_config_presets.rs#L228).

**Review by al3mart — COMMENTED** [review]
Wouldn't have pushed back on including `Litep2p`. But appreaciate the prio.

Just left a comment regarding that we could add the genesis configs in the runtime with the changes coming from [#2714](https://github.com/paritytech/polkadot-sdk/pull/2714). More as a reminder than anything else, as this might want to be done once we have finish working on the chain of upgrades.

Though, it is something I believe we want to do anyway.

File: runtime/devnet/src/lib.rs (hunk 1 above)
Comment (chungquantin) [reply runtime/devnet/src/lib.rs:953]: Just notice that part, I will fix it. Thanks for reporting it

Comment (chungquantin) [reply runtime/devnet/src/lib.rs:953]: Created an issue: https://github.com/r0gue-io/pop-node/issues/196

File: runtime/devnet/build.rs
Comment (peterwht) [inline runtime/devnet/build.rs:1]: Wasm builder simplification: https://github.com/paritytech/polkadot-sdk/pull/4177

File: runtime/testnet/build.rs
Comment (peterwht) [inline runtime/testnet/build.rs:1]: https://github.com/paritytech/polkadot-sdk/pull/4177

**Review by peterwht — APPROVED** [review]
Approving! Added some "nice-to-have" comments for the build.rs. You can postpone to later PRs if it's easier for git rebasing etc.

**Comment by peterwht** [comment]
[sc-918]

**Review by al3mart — APPROVED** [review]

**Pull request merged**